### ISSUE TITLE
Errors Booting "Alpha" Releases

### ISSUE THREAD

**Issue by chad-thompson** [coreos/coreos-vagrant#244]
I've been poking around a bit with the project and have found an issue 'out of the box'.  I've tried to fix the problem, but I'm a systemd newbie and haven't found 'the answer' yet.  The basic issue:  a system service is attempting to start the cloud-init service prior to Vagrant copying the file (with the file provisioner) to the virtual machine - but only in the 'alpha' channel (which is the default for the project in the Vagrantfile).  Here's the issue:

`Vagrant::Errors::VagrantError: The following SSH command responded with a non-zero exit status.
Vagrant assumes that this means the command failed!
systemctl start system-cloudinit@-var-tmp-hostname.yml.service
Stdout from the command:
Stderr from the command:
Job for system-cloudinit@-var-tmp-hostname.yml.service failed because a configured resource limit was exceeded. See "systemctl status system-cloudinit@-var-tmp-hostname.yml.service" and "journalctl -xe" for details.`

The error from the systemctl status command:

`Jun 13 22:11:46 localhost systemd[1]: system-cloudinit@-var-tmp-hostname.yml.service: Failed to run 'start' task: Invalid argument`

Now if you change the update_channel to be 'stable' in configuration, then it all works like a charm:  the machine boots and starts up properly.

`● system-cloudinit@-var-tmp-hostname.yml.service - Load cloud-config from /var/tmp/hostname.yml
   Loaded: loaded (/usr/lib64/systemd/system/system-cloudinit@.service; static; vendor preset: disabled)`

TL;DR - if you're using the project out of the box to play with CoreOS, change the update_channel from 'alpha' to 'stable' and things will work.

**Comment by crawford** [comment]
Duplicate of https://github.com/coreos/bugs/issues/382.
